pymc/gp/cov.py

```diff
@@ -812,6 +824,52 @@ def full(self, X, Xs=None):
     def diag(self, X):
         X, _ = self._slice(X, None)
         return self.cov_func(self.w(X, self.args), diag=True)
+    
+
+class WrappedPeriodic(Covariance):
+    r"""
+    Wrap a stationary covariance function to make it periodic.
+
+    This is done by warping the input with the function
+
+    .. math::
+        \mathbf{u}(x) = \left(
+            \mathrm{sin} \left( \frac{2\pi x}{T} \right),
+            \mathrm{cos} \left( \frac{2\pi x}{T} \right)
+        \right)
+
+    Parameters
+    ----------
+    cov_func: Stationary
+        Base kernel or covariance function
+    period: Period
+    """
+
+    def __init__(
+        self,
+        input_dim: int,
+        cov_func: Stationary,
+        period,
+        active_dims: Optional[Sequence[int]] = None,
+    ):  
```

Comment:
Can `WrappedPeriodic` take `input_dim` and `active_dims` from `cov_func`?  That way these don't need to be repeated.

Reply:
That makes sense. My only concern would be it is then the only `Covariance` subclass which doesn't take those params on init.